src/Resources/config/services.yml

```diff
@@ -1,9 +1,3 @@
 services:
-    _instanceof:
-        Contao\CoreBundle\Framework\FrameworkAwareInterface:
-            calls:
-                - ["setFramework", ["@contao.framework"]]
-
-        Symfony\Component\DependencyInjection\ContainerAwareInterface:
-            calls:
-                - ["setContainer", ["@service_container"]]
```

Comment:
Using these interfaces is not best practise any more I think? Thus I have removed them. These lines can be seen in a lot of Contao extensions out there - even though nobody ever actually uses those interfaces …